.github/workflows/deploy-to-control-plane-review.yml

```
@@ -1,85 +1,72 @@
-  PR_NUMBER: ${{ github.event.issue.number }}
+  APP_NAME: qa-react-webpack-rails-tutorial-pr-${{ github.event.pull_request.number || github.event.issue.number }}
+  CPLN_ORG: ${{ secrets.CPLN_ORG_STAGING }}
+  CPLN_TOKEN: ${{ secrets.CPLN_TOKEN_STAGING }}
+  PR_NUMBER: ${{ github.event.pull_request.number || github.event.issue.number }}
 
 jobs:
-  deploy-to-control-plane-review:
-    if: ${{ github.event_name != 'issue_comment' || (github.event.comment.body == '/deploy-review-app' && github.event.issue.pull_request) }}
+  deploy:
+    if: |
+      github.event_name == 'pull_request' || 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)